docs/content/docs/projects/BuildChecks.md

```diff
@@ -17,24 +17,7 @@ plugins {
 }
 ```
 
-In the `settings.gradle`:
-
-```groovy
-pluginManagement {
-    repositories {
-        jcenter()
-    }
-    resolutionStrategy {
-        eachPlugin {
-            String pluginId = requested.id.id
-            if (pluginId.startsWith("com.avito.android")) {
-                def artifact = pluginId.replace("com.avito.android.", "")
-                useModule("com.avito.android:$artifact:$avitoToolsVersion")
-            }
-        }
-    }
-}
-```
+{{%project-setup%}}
```

Comment:
`project` is a common term. What about more specific name? For instance, `plugins-setup`.

Reply:
renamed